src/edu/csus/ecs/pc2/core/execute/SandboxInput.java

```diff
@@ -15,11 +15,22 @@
  */
 public class SandboxInput {
 
-    private static final String SANDBOX_PROPERTIES_FILENAME = "sandbox.properties";
+    /**
+     * Judges data file name, aka sumit.in
+     */
+    public static final String DATAFILENAME_KEY = "datafilename";
+    
+    /**
+     * Source file name, aka Sumit.java
+     */
```

Comment:
This is a comment regarding a very very very minor issue, BUT,  in the spirit of accuracy:  "aka" means "Also Known As".  So what the above comment says is that "This is the source file name, also known as Sumit.java".  I think what is meant is "e.g." (short for "exempli gratia") which means "For example".